Makefile

```diff
@@ -791,6 +791,10 @@ pytest_pypi_wheel_cml_no_flaky:
 pytest_pip_cml:
 	./script/make_utils/pytest_pypi_cml.sh
 
+.PHONY: pytest_pip_cml_no_flaky # Run tests (except flaky ones) using PyPI Concrete ML
+pytest_pip_cml_no_flaky:
```

Comment:
Shouldn't this be changed to pypi in the name as the one above for coherence?

Reply:
yes definitively